ci/jenkins/testkomodo.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -xe
 PROJECT=libres
+RELEASE_PATH=${KOMODO_ROOT}/${RELEASE_NAME}
```

Comment:
At line 12 where RELEASE_PATH is used - ${RELEASE_NAME} is appended once more. Is this correct?

Reply:
Yes, because that's a file in which you'll find a yaml with versions